frontend/src/components/IOCsTable/IOCsTable.tsx

```diff
@@ -126,13 +129,42 @@ const IOCsTable: React.FC<IOCsTableProps> = ({ iocs }) => {
                   <TableCell style={{ fontFamily: "monospace" }}>
                     {row.cells[1].value}
                   </TableCell>
+                  <TableCell>
+                    <Tag
```

Comment:
If we move it to options menu on the right side we don't need this Tag like here: 
![image](https://github.com/user-attachments/assets/2ab91fa0-0b86-4caa-8024-fe348168dafe)

https://react.carbondesignsystem.com/?path=/story/components-datatable-toolbar--with-overflow-menu